tests/migration/online_migration/zypper_migration.pm

```diff
@@ -54,7 +54,7 @@ sub run {
         script_run("(zypper migration $option; echo ZYPPER-DONE) |& tee /dev/$serialdev", 0);
     }
     # migration process take long time
-    my $timeout = (is_leap_migration && (check_var('DESKTOP', 'kde'))) ? 10800 : 7200;
+    my $timeout = (is_leap_migration) ? 18000 : 7200;
```

Comment:
Add some comments for the reason we set 5h for leap migration.

Reply:
added && thanks